lib/index.js

```diff
@@ -587,4 +611,4 @@ function getNcurc({ configFileName, configFilePath, packageFile } = {}) {
   return result ? { ...result, args } : null
 }
 
-module.exports = { run, getNcurc, getPeerDependencies, ...vm }
+module.exports = { run, getNcurc, getPeerDependencies, mergeOptions, ...vm }
```

Comment:
We don't want to add `mergeOptions` to the public api just for testing. Instead, define it in its own module in `/src/lib` and import it directly in the test.

Reply:
OK